src/controllers/expenses.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const expensesServices = require('../services/expenses');
+
+const getAll = (req, res) => {
+  const expenses = expensesServices.filterExpenses(req.query);
+
+  res.send(expenses);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+  const foundExpenses = expensesServices.getById(id);
+
+  if (!foundExpenses) {
+    res.status(404).send('Expenses not found');
```

Comment:
you just to find one
```suggestion
    res.status(404).send('Expense not found');
```

Reply:
Done